app/controllers/distributions_controller.rb

```diff
@@ -150,11 +150,11 @@ def insufficient_error_message(details)
   end
 
   def send_notification(org, dist, subject: 'Your Distribution')
-    PartnerMailerJob.perform_async(org, dist, subject) if Flipper.enabled?(:email_active)
+    PartnerMailerJob.perform_now(org, dist, subject) if Flipper.enabled?(:email_active)
```

Comment:
`:perform_async` is a method in `Sidekiq::Worker`, but we're referencing `ActiveJob` directly now